### ISSUE TITLE
Images and Markdown 

### ISSUE THREAD

**Issue by ClarissaSpain** [Montana-Media-Arts/120_CreativeCoding#76]
My biggest issue has been adding images. It feel like it's a hit or miss, because I'm not able to see it in the Atom preview and it seemed like it was a broken link. After some further research I was able to upload my picture onto [amazonaws](https://aws.amazon.com/console/), there the website stores the image and creates an URL. I make the image public, and with the URL I can then copy and paste into my markdown file. My next issue has been sizing and that's where the html that I have provided below comes in.
```html
<img src="https://s3-us-west-1.amazonaws.com/gitwebsite/Cactus-Logo.png"
alt="Baby Cactus" style="width: 150px;"/>
```
But then I realized it didn't effect how it looked on the web, simply in the preview. So I went back to my photo and compressed the 1000x1000 px image to a 150x150 px and reuploaded it again to Amazon S3 service. Then did the regular markdown ```![alt](url.png)``` It sort of worked but I genuinely feel like there's an easier way to do this? Can I change the placement of my picture? Can I take a large picture and through code compress it into a smaller file size?

**Comment by cloninger-carrie** [comment]
Make sure that when you type in the code linking to your picture that it has the correct file extension. The example in the online content uses .jpg. Mine, however, was a .PNG. You can find this out by looking at the picture file information.

**Comment by ReadReedRed** [comment]
I think you may have gone a bit above and beyond what you were expected to do in this case; if you include the image in the folder you made for this week's homework (within your GitHub repository folder), commit it in GitHub, and push it, you should just be able to include it in your Atom with just the file name, not a URL. 

For example, I saved my screenshot to that folder as a .jpg called A2_Text_Editor, so I attached it to Atom with this line: 
![Image of my editor](A2_Text_Editor.jpg)

Let me know if you need any clarification on that! Hope this helped.

**Comment by MarkLannenUM** [comment]
You mentioned that you weren't able to preview images in atom. Is that correct? If so, you can use the shortcut ctr + shift + m, in atom and you should then see a preview of the markdown.

**Comment by ClarissaSpain** [comment]
@ReadReedRed  I do agree with you. I have a this tendency to make things more difficult than necessary.  I did commit a file to the repository folder but I didn't try adding it normally into Atom. So I took extra steps, I guess in a sense I got to the same finish line, but it just took a little bit longer. I guess that's what makes coding really interesting because there are so many different paths that a person can take. Thank you, I needed clarification that there was an easier way out there. 

@MarkLannenUM  Wow, this is a great shortcut! Thank you for letting me know.
